service_identity/src/test/kotlin/org/thechance/service_identity/domain/usecases/validation/UserValidationUseCaseTest.kt

```diff
@@ -7,7 +7,7 @@ import org.junit.jupiter.api.Test
 
 class UserValidationUseCaseTest {
 
-    private val userValidationUseCase = UserValidationUseCase()
+    private val userValidationUseCase = ValidateUserInfoUseCase()
 
     @Test
     fun should_ReturnFalse_When_TheUsernameIsBlank() {
```

Comment:
Ensuring consistent naming of test functions, compare with another file

Reply:
@Youssef-Ramadan-Y Have you completed the changes?